packages/@azure/servicebus/data-plane/test/namespace.spec.ts

```diff
@@ -325,3 +328,44 @@ describe("Errors when send/receive to/from non existing Queue/Topic/Subscription
     should.equal(errorWasThrown, true, "Error thrown flag must be true");
   });
 });
+
+describe("Test createFromAadTokenCredentials", function(): void {
```

Comment:
Let's add 2 more test cases for error scenarios when `host` and `credentials` are not passed or invalid values are passed